content/Watsonx/WatsonxData/109.md

```diff
@@ -16,7 +16,7 @@ The TechZone watsonx.data environment comes pre-configured with presto engine, i
 
 3. In the **Add bucket** pop-up window, select/enter the following pieces of information
 
-> **Note:** If you have an IBM Cloud account, you can create your own bucket and use it in this lab. You can use the following bucket but make sure, you prefix your schema with your initials.
+> **Note:** If you have an IBM Cloud account, you can create your own bucket and use it in this lab. You can also use the following bucket but make sure, you prefix your schema name with your initials to avoid object conflict issue in bucket. 
```

Comment:
Minor: Can we update the last part to the following?

> You can also use the following bucket provided for your workshop, but make sure you prefix your schema name with your initials to avoid naming conflicts in the COS bucket.

Reply:
Actually it's probably best to add this note below where the schema name is actually added.